test/helpers/kubectl.ts

```diff
@@ -102,6 +108,31 @@ export async function getPodLogs(podName: string, namespace: string): Promise<an
   return logsOutput.stdout;
 }
 
+export async function waitForPodsReady(partialName: string, namespace: string): Promise<void> {
```

Comment:
this seems a bit funky, can we be a bit more clear here? why do we need a partial match followed by waiting for it to be ready? can't we wait for a condition on some deployment to be 1/1 or something like that?